test/plug/conn_test.exs

```diff
@@ -517,14 +517,14 @@ defmodule Plug.ConnTest do
       ~S[value for header "x-sample" contains control feed (\r) or newline (\n): "value\rBAR"]
 
     assert_raise Plug.Conn.InvalidHeaderError, message, fn ->
-      put_resp_header(conn(:get, "foo"), "x-sample", "value\rBAR")
+      put_resp_header(conn(:get, "/foo"), "x-sample", "value\rBAR")
```

Comment:
All the tests that needed fixing raised errors early because of the new constraint.
Adding the slash in the correct place makes sure they still test what was intended.